build_helper.sh

```diff
@@ -224,18 +240,11 @@ function setup_folly() {
   cd "$FOLLY_BUILD_DIR" || exit
 
   # check for environment variable. If
-  if [[ -z "${MVFST_FOLLY_USE_JEMALLOC-}" ]]; then
-    cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo         \
-      -DCMAKE_PREFIX_PATH="$FOLLY_INSTALL_DIR"      \
-      -DCMAKE_INSTALL_PREFIX="$FOLLY_INSTALL_DIR"   \
-      ..
-  else
-    cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo         \
-      -DCMAKE_PREFIX_PATH="$FOLLY_INSTALL_DIR"      \
-      -DCMAKE_INSTALL_PREFIX="$FOLLY_INSTALL_DIR"   \
-      -DFOLLY_USE_JEMALLOC=0                        \
-      ..
-  fi
+  cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo         \
+    -DCMAKE_PREFIX_PATH="$FOLLY_INSTALL_DIR"      \
+    -DCMAKE_INSTALL_PREFIX="$FOLLY_INSTALL_DIR"   \
+    ${CMAKE_EXTRA_ARGS}                           \
```

Comment:
Internal linter is complaining about not using double quotes for the variable expansion, can you fix that for all instances of this?